client/src/components/Views/TripSidePanelSingle.js

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useMemo, useState } from 'react'
+import { TeaCup } from './TeaCup'
+import { Button } from '../common'
+import { setAppView } from '../../redux/reducers/viewSlice'
+import { AppView } from '../../constants/enums'
+import { useDispatch, useSelector } from 'react-redux'
+import TripViewJson from '../../temp/tripViewData.json'
+import { getColorName } from '../../utils/translateTailwindColors'
+
+export function TripSidePanelSingle() {
+  const activeDayNumber = useSelector((state) => state.view.activeDayNumber)
+  const dispatch = useDispatch()
+  const [dayDetails, setDayDetails] = useState()
+
+  useEffect(() => {
+    // TODO this logic needs to fetch the right trip ID when we have real data
+    setDayDetails(TripViewJson[activeDayNumber - 1])
+  }, [activeDayNumber])
```

Comment:
should we reset activeDayNumber back to 1 when activeTripId is changed (when a user click on a new trip)

Reply:
oh good idea

Reply:
added change to reducer